includes/class-gp-translation-helpers.php

```diff
@@ -72,6 +72,18 @@ public function __construct() {
 
 		add_filter( 'gp_translation_row_template_more_links', array( $this, 'translation_row_template_more_links' ), 10, 5 );
 		add_filter( 'preprocess_comment', array( $this, 'preprocess_comment' ) );
+		add_filter(
+			'gp_tmpl_load_locations',
+			function( $locations, $template, $args, $template_path ) {
+				if ( 'translation-row-editor-meta-status' === $template ) {
```

Comment:
Let's always add
````
$locations[] = dirname( dirname( __FILE__ ) ) . '/templates/';
```

in addition and remove the paths from all of our `gp_tmpl_load` so that they can be easily overriden.